main.js

```diff
@@ -1865,6 +1961,35 @@ function _select_Gateway() { // Check for Gateway used to
         },
     });
 
+    // expand recursive tasklist
+    for (var professionIdx in definedTask) {

```

Comment:
If to be precise it's O(n) where n is the number of total levels in all the profiles. 
Or O(nm) where m is the number of items in a level.
If to be practical it's maximum 13*10*25*5, and sequential reading.
This is not important,